src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -351,8 +351,6 @@ private void ProcessTraceroute(string targetNameOrAddress)
             IPAddress hopAddress;
```

Comment:
@vexx32, your last commit had 51 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At D:\a\1\s\test\powershell\Language\Operators\NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Tests for $ErrorView.NormalView tests.Error shows up when using strict mode
```powershell
Expected like wildcard '*Oops!*' to match '
', but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Formatting\ErrorView.Tests.ps1: line 82
82:             $e | Should -BeLike '*Oops!*'
```
TabCompletion.Should complete dotnet method with null conditional operator
```powershell
Expected exactly 'ToString(', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 38
38:         $res.CompletionMatches[0].CompletionText | Should -BeExactly 'ToString('
```